src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -901,15 +905,18 @@ export class TerminalService implements ITerminalService {
 			quickPickItems.push(...configProfiles.map(e => this._createProfileQuickPickItem(e)));
 		}
 
-		// Add contributed profiles
-		if (type === 'createInstance') {
-			for (const contributed of this._terminalContributionService.terminalProfiles) {
-				const icon = contributed.icon ? (iconRegistry.get(contributed.icon) || Codicon.terminal) : Codicon.terminal;
-				quickPickItems.push({
-					label: `$(${icon.id}) ${contributed.title}`,
-					profile: contributed
-				});
-			}
+		for (const contributed of this._terminalContributionService.terminalProfiles) {
+			const icon = contributed.icon ? (iconRegistry.get(contributed.icon) || Codicon.terminal) : Codicon.terminal;
+			quickPickItems.push({
+				label: `$(${icon.id}) ${contributed.title}`,
+				profile: {
+					extensionIdentifier: contributed.extensionIdentifier,
+					title: contributed.id,
+					icon: contributed.icon,
+					id: contributed.id,
+					isSplitTerminal: contributed.isSplitTerminal
```

Comment:
Again split terminal this looks wrong being here, whether it's a split terminal depends on alt below